docs-src/security/data-converter.md

```diff
@@ -0,0 +1,15 @@
+---
+id: data-converter
+title: Data Converter
+sidebar_label: Data Converter
+description: Use a Data Converter with a custom Payload Codec to encode and decode your data.
+tags:
+  - guide-context
+---
+
+Each Temporal SDK provides a [Data Converter](/concepts/what-is-a-data-converter) that can be customized with a custom [Payload Codec](/concepts/what-is-a-payload-codec) to encode and secure your data.
+
+For details on what data can be encoded, how to secure it, and what to consider when using encryption, see [Data encryption](/prod-readiness-context/data-encryption).
```

Comment:
@aarohib this link doesn't work. Directs to `https://production-readiness/develop#data-encryption`

Reply:
dang; fixing.